.mailmap

```diff
@@ -1095,6 +1095,8 @@ Robert Dougherty-Bliss <robert.w.bliss@gmail.com>
 Robert Johansson <jrjohansson@gmail.com>
 Robert Kern <robert.kern@gmail.com>
 Robert Pollak <robert.pollak@posteo.net> Robert Pollak <robert.pollak@jku.at>
+Robin Richard <raisin@ecomail.fr> robinechuca <raisin@ecomail.fr>
+Robin Richard <raisin@ecomail.fr> robinechuca <61911191+robinechuca@users.noreply.github.com>
```

Comment:
These lines should not be added to the .mailmap file.

Reply:
A bit strange, because I didn't manually add these lines. Maybe I forgot to delete them when I was resolving the merge conflict.

Reply:
Where should I run `python bin/mailmap_check.py ` , I can't see such a command in the `bin` directory.

Reply:
Personally I would like to get my work in before the release, so I'll make a separate pull request if I can't workaround the failing authors test soon.

Reply:
This PR was created before the `bin/mailmap_check.py` script was added. If you delete these two lines and rebase to current master and squash the commits then it should be fine. You might find it easier just to start a fresh PR from current master though.